src/core/icons/fileIcons.ts

```diff
@@ -3279,5 +3279,41 @@ export const fileIcons: FileIcons = {
       name: 'metro',
       fileNames: ['metro.config.js', 'metro.config.json'],
     },
+    {
+      name: 'bashly',
+      patterns: {
+        'src/bashly': FileNamePattern.Yaml,
+      },
+    },
+    {
+      name: 'bashly-settings',
+      clone: {
+        base: 'bashly',
+        color: 'gray-300',
+        lightColor: 'gray-800',
+      },
+      patterns: {
+        'bashly-settings': FileNamePattern.Yaml,
+      },
+      light: true,
+    },
+    {
+      name: 'bashly-settings',
+      clone: {
+        base: 'bashly',
+        color: 'gray-300',
+        lightColor: 'gray-800',
+      },
+      patterns: {
+        settings: FileNamePattern.Yaml,
+      },
+      light: true,
+      enabledFor: [IconPack.Bashly],
```

Comment:
Do we actually need it two times? If I don't miss something, then the first config here can be removed, as we will rely on the icon pack setting?
```suggestion
      name: 'bashly-settings',
      clone: {
        base: 'bashly',
        color: 'gray-300',
        lightColor: 'gray-800',
      },
      patterns: {
        settings: FileNamePattern.Yaml,
      },
      light: true,
      enabledFor: [IconPack.Bashly],
```

Reply:
Well, according to my tests, this works and makes a difference.

I decided to do it this way so that the icon will _always_ be shown for the `bashly-settings.yml` file, which is specific to Bashly, regardless of whether the icon package is active or not.

As for the `settings.yml` file, which may or may not refer to Bashly since it has a generic name, the icon will only be displayed if the user explicitly wants it by enabling the icon package.

Does that make sense?

Reply:
Yes, that makes sense. I didn’t recognize the different patterns initially 🙈